package/MDAnalysis/analysis/pcz.py

```diff
@@ -0,0 +1,481 @@
+"""
+A PCA module for MDAnalysis
+==========================================================================
```

Comment:
+A PCA module for MDAnalysis
+==========================================================================

should be
+A PCA module for MDAnalysis
+=======================